deploy/group_vars/amibuilder/main.yml

```diff
@@ -67,6 +67,7 @@ django_project_settings:
   RAVEN_CONFIG:
     dsn: "{{ vault_sentry_dsn }}"
     environment: "{{ sentry_environment }}"
+    release: '0.5.1'
```

Comment:
This bit adds the version number to the configuration for sentry. The available options are documented [here](https://docs.sentry.io/clients/python/advanced/#client-arguments).